CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 8.16.1
+  - Provides a guidance in logs when plugin version mismatches with connected Elasticsearch `major.minor` version [#253](https://github.com/elastic/logstash-filter-elastic_integration/pull/253)
+  - Embeds Ingest Node components from Elasticsearch 8.16
+  - Compatible with Logstash 8.15+
```

Comment:
Given that 8.16.0 is already 8.15+ compatible should we repeat also in this patch version?

Reply:
Same opinion: https://github.com/elastic/logstash-filter-elastic_integration/pull/260#discussion_r1932384212